### PR TITLE
Backend configuration with support tools

### PR DESCRIPTION
**Pull request by mateuszzalewski011**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/functiona_feature_request.md
Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:2]:
```suggestion
name: Feature request
```

File: .github/ISSUE_TEMPLATE/functiona_feature_request.md
Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:13]:
```suggestion

```

Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:13]:
```suggestion
```

File: test/__init__.py
Comment (Argmaster) [inline test/__init__.py:1]:
```suggestion
"""Epseon GUI test suite root."""
```

**Review by Argmaster — CHANGES_REQUESTED** [review]

**Comment by Argmaster** [comment]
I'd be great to have those fixed before next meeting, maybe even merged before.

File: .dockerignore
Comment (Ave44) [inline .dockerignore:1]: If we'll change name of our dockerfiles then we should add rule `*.dockerfile`

**Review by Ave44 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/functiona_feature_request.md
Comment (Argmaster) [inline .github/ISSUE_TEMPLATE/functiona_feature_request.md:3]: You have messed up header structure.

File: .pre-commit-config.yaml
Comment (Argmaster) [reply .pre-commit-config.yaml:4]: @mateuszzalewski011 please change this to `v3.0.3`

Comment (Argmaster) [reply .pre-commit-config.yaml:4]: Okay, but it still shows as `v2.7.1` in review, am I missing something?

**Comment by Argmaster** [comment]
Any chance to get this merged? I am still waiting for response on Prettier version.

File: .pre-commit-config.yaml
Comment (Ave44) [reply .pre-commit-config.yaml]: version updated

**Review by Ave44 — DISMISSED** [review]

**Review by Argmaster — DISMISSED** [review]

**Review by Argmaster — APPROVED** [review]

**Review by Ave44 — APPROVED** [review]

**Pull request merged**
